### PR TITLE
Add some parameter checks

### PR DESCRIPTION
**Pull request by shansfolder**
As we discussed in the meeting.

### CODE REVIEW

File: expan/core/binning.py [hunk 1]
```diff
@@ -132,12 +132,14 @@ def __init__(self, categories):
         Constructor for representation of a categorical bin.
         :param categories: list of categorical values that belong to this bin
         '''
-        if type(categories) is not list:
-            raise ValueError("args for categorical bin is a list of categorical values.")
+        if type(categories) is list:
```
Comment (gbordyugov) [inline expan/core/binning.py]: what happens if it isn't list?

we could try just to cast to set and catch an exception

Comment (shansfolder) [reply expan/core/binning.py]: It is in the next lines. If it's neither list nor set, we will raise an error.

Comment (gbordyugov) [reply expan/core/binning.py]: Yeah, I'm just saying that
```
categories = set(categories)
```
would do the same job as four lines (135 through 138). Please correct me if I'm wrong.

**Review by mkolarek — APPROVED** [review]

File: expan/core/binning.py (hunk 1 above)
Comment (shansfolder) [reply expan/core/binning.py]: Right..Done!

**Pull request merged**